### DISCUSSION TITLE
Menu content type does not change its language

### DISCUSSION THREAD

**Discussion by aleko279** [OrchardCMS/OrchardCore#6941] [Q&A] [answered]
Hello, I am creating Content Culture Picker for my project . When I switch language it works for every content of website except the menu. Menu does not change its language while I also add localization to Menu type. What can I do in order to fix this?

**Comment by deanmarcussen** [comment, marked as answer]
Here is some code - credit to @jptissot for a technique to load different menus based on the `Alias`. It assumes that the `Alias` has been coded with the culture name

```
{% assign menu = "alias:menu-" | append: Culture.Name %}
        <nav class="navbar main-nav navbar-expand-lg">
            {% shape "Menu", alias: menu %}
        </nav>
```

**Reply by DotCat1985** [reply]
Is it possibile to create localized menus without using alias?

**Reply by hishamco** [reply]
Menus can't be localized at the time, please refer to #4466
